src/js/components/LanguageWrapper.js

```diff
@@ -116,7 +116,7 @@ class LanguageWrapper extends Component {
         <div style={{ position: 'relative' }}>
           <LanguageSelectBanner lang={lang} path={this.props.match.params.path || ''}/>
           <Header />
-            <section className="coa-main">
+            <section>
```

Comment:
should this be `<main className="coa-main" role="main" id="main">`? For a11y?

Reply:
YES